docs/references/cmcspec.mdx

```diff
@@ -0,0 +1,552 @@
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# Cycles Minting Canister (CMC) API
+
+<MarkdownChipRow labels={["Reference"]} />
+
+The Cycles Minting Canister (CMC) is a system canister on the Internet Computer responsible for converting ICP tokens into **cycles**, tracking the total cycles minted, and providing configuration and exchange rate data. It supports:
+
+-   Topping up existing canisters.
+-   Creating new canisters.
+-   Minting and depositing minted cycles into cycle ledger-managed accounts.
+-   Fetching exchange rates and subnet configuration.
+
+The CMC is governed solely by the [NNS (Network Nervous System)](https://internetcomputer.org/nns/) and receives configuration updates through proposals.
+
+## Configuration parameters
+
+-   **Conversion rate source**: The ICP/XDR exchange rate is **pushed** into the CMC by a dedicated [exchange rate canister](/references/exchange-rate-canister). This data reflects live market pricing and determines the ICP→cycles conversion.
+-   **Governance control**: All operations of the CMC — including upgrades, configuration, and authorized callers — are managed through NNS proposals.
+## API endpoints
+
+The CMC exposes a set of core methods for converting ICP into cycles and interacting with subnet configuration. These include:
+
+-   `notify_create_canister`: Processes an ICP payment by minting cycles and using them to create a new canister, assigning control to the specified principal and applying optional settings.
```

Comment:
does this method also burn the ICP tokens or has the burning already happened before the notification?
I thought maybe to understand the flow better, it would be useful to say in a sentence what precondition this has / what usually happened before this is executed?

Reply:
Ah I now see we go into the details below, so resovling this comment!